src/client/app/effects/snippet.ts

```diff
@@ -34,9 +34,9 @@ export class SnippetEffects {
                 .map((snippet: ISnippet) => ({ snippet, mode }))
                 .filter(({ snippet }) => !(snippet == null))
                 .mergeMap(({ snippet, mode }) => this._massageSnippet(snippet, mode, isViewMode))
```

Comment:
I think "massageSnippet" is no longer the right name.  You're not massaging the snippet, you're instead importing it.  Maybe "_importHelper"?